plutus-core/cost-model/test/TestCostModels.hs

```diff
@@ -247,13 +174,13 @@ testPredictThree haskellModelFun modelR1 = propertyR $ do
   modelH <- lift $ haskellModelFun modelR
   let
     predictR :: MonadR m => CostingInteger -> CostingInteger -> CostingInteger -> m CostingInteger
-    predictR x y _z =
+    predictR x y z =
       let
         xD = fromIntegral x :: Double
         yD = fromIntegral y :: Double
-        -- zD = fromInteger z :: Double
+        zD = fromIntegral z :: Double
       in
-        msToPs . fromSomeSEXP <$> [r|predict(modelR_hs, data.frame(x_mem=xD_hs, y_mem=yD_hs))[[1]]|]
+        msToPs . fromSomeSEXP <$> [r|predict(modelR_hs, data.frame(x_mem=xD_hs, y_mem=yD_hs, z_mem=zD_hs))[[1]]|]
```

Comment:
How did this work before???

Reply:
It happens that all of our current three-argument builtins are effectively constant-cost, so the R code was never looking at the third argument (not sure why that was commented out). It still doesn't look at it, but I put it back just in case we ever have a builtin where the third argument is important.